ios/KaMPKitiOS/AppDelegate.swift

```diff
@@ -11,25 +11,25 @@ import shared
 
 @UIApplicationMain
 class AppDelegate: UIResponder, UIApplicationDelegate {
-    
+
     var window: UIWindow?
 
     // Lazy so it doesn't try to initialize before startKoin() is called
+    // swiftlint:disable force_cast
```

Comment:
Is there any way to globally opt-out a specific API so we don't need to do this each time?